### PR TITLE
[Suggestion] README with DAG file

### PR DESCRIPTION
**Pull request by Mxrcon**
# Poposal
Hello! I'd like to contribute with this project suggesting the enhancement of the  `README ` file, i took some time writing a simple, but clearer that could help new users on the basic usage of this pipeline, 

# Main Topics
I tried to clarify the basic features of this pipeline on the README file, I hope it will be helpful for new users as they could run it with the reading of this file.

# Ending
I'm your disposal, fell free to ask me anything or criticize any changes if you wish to.

Hope we have a merge!
Regards, Davi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,111 @@
...
+This project has the `-stub-run` feature, that can be used for testing propouse, it can be used on `Tower` with the Advanced settings on launch. You can also test it locally, using the following command:
+
+```
+bash data/mock_data/generate_mock_data.sh
+nextflow run main.nf \
+		 -params-file stub_params.yaml \
+		 -stub-run
+``` 
+
+## Project structure
```
Comment (abhi18av) [inline README.md:69]: I think that since the source code is available for everyone, this section isn't needed.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,111 @@
...
+$READ = STR, replace for your reads location. You can write using READ_{1,2}.fastq.gz or READ_1.fastq.gz READ_2.fastq.gz 
+
+$GBK = STR, replace for your reference gbk lcation.
+
+$OUDIR = STR, replace for the name of your desired output directory.
+
+## Configuration Profiles.
+
+You can use diferent profiles for this pipeline, based on the computation enviroment at your disposal. Here are the Avaliable Profiles:
+
```
Comment (abhi18av) [inline README.md:47]: Need to add a local profile as well.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,111 @@
...
+```
+
+3. Run it!
+
+
+```
+	nextflow run main.nf --reads $READ --gbkFile $GBK --outdir $OUTDIR
+
+```
```
Comment (abhi18av) [inline README.md:36]: There's technically no need to download and then run the workflow, anyone can simply do 

```
nextflow run https://github.com/bioinformatics-lab/camila_sao_paulo_nf --reads $READ --gbkFile $GBK --outdir $OUTDIR
```

**Review by abhi18av — CHANGES_REQUESTED** [review]
Hey Davi,

This PR looks good, however there are some minor change requests before I can merge this.

File: README.md (hunk 2 above)
Comment (Mxrcon) [reply README.md:47]: ok! added

File: README.md (hunk 3 above)
Comment (Mxrcon) [reply README.md:36]: That's true, I'm adding it to the readme

**Comment by Mxrcon** [comment]
Thank you 😃! 🎉

**Comment by abhi18av** [comment]
Eu que agradeço @Mxrcon  😊

**Pull request merged**